actors/builtin/market/market_state_test.go

```diff
@@ -0,0 +1,30 @@
+package market_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/builtin/market"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+	"github.com/filecoin-project/specs-actors/support/mock"
+)
+
+func TestConstruction(t *testing.T) {
+	rt := mock.NewBuilder(context.Background(), address.Undef).Build(t)
+
+	t.Run("initializes all fields", func(t *testing.T) {
```

Comment:
This test would be more robust if it verified fields containing CID's were the expected value, e.g.
```go
emptyMap, err := adt.MakeEmptyMap(rt.Store())
if err != nil {
    return nil, err
}
assert.Equal(r, emptyMap, state.EscrowTable)
/* ... similar assertions for EmptyArrarys ...  */
```